specification/notebooks/resource-manager/Microsoft.Notebooks/preview/2019-10-11-preview/example/NBResourceCreate.json

```diff
@@ -0,0 +1,26 @@
+{
+  "parameters": {
+    "resourceName": "testResource",
+    "resourceGroupName": "testRP",
+    "subscriptionId": "91e9f7ac-9c2c-4cd2-af61-1c5b57ec6c51",
+    "resourceReuest": {
+      "location": "southcentralus",
+      "properties": {
+      },
```

Comment:
you need to pass the properties in PUT

Reply:
I have linked a prior[ PR](https://github.com/Azure/azure-rest-api-specs-pr/pull/1086) that has been approved.